### ISSUE TITLE
Color Palette Demo

### ISSUE THREAD

**Issue by jcuenod** [hyperfocalHQ/pollen#98]
Hey, I just discovered this library and it looks awesome.

One thing I noticed in the docs is that the docs show hex values for the color palette. It would be great to see a swatch map of the colors on that page.

**Comment by madeleineostoja** [comment]
Yep they actually used to have that but it was removed in the recent major version, since the colour palette doubled in size. Will add back in when I get time!
